client/components/vertical-menu/style.scss

```diff
@@ -41,3 +49,6 @@
 	margin-bottom: -4px;
 }
 
+.vertical-menu__items__social-icon .social-logo {
+	fill: $gray;
```

Comment:
Is there a reason why the social logo normal state has a different shade of grey than its menu item (see line 14, `$gray-text`)?